### PR TITLE
mass_of_film_deposited_during_electrolysis

### PR DESCRIPTION
**Pull request by Zulusssss**
add law and tests

### CODE REVIEW

File: symplyphysics/laws/chemistry/mass_of_film_deposited_during_electrolysis.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+# Description
+## Electrolysis is a physico-chemical process consisting in the release of components of dissolved substances or other substances on the electrodes,
+## which are the result of secondary reactions on the electrodes, which occurs when an electric current passes through a solution or melt of an electrolyte.
+## Current output is the proportion of electric current spent on the passage of the target electrochemical reaction.
+
+## Law is: m =  J * M * B * t / (Z * q * Na), where
+## m - mass of film,
+## J - the current in the electrolyte,
+## M - molar mass of the metall atom,
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/mass_of_film_deposited_during_electrolysis.py:14]:
```suggestion
## M - molar mass of the atom of metal,
```

File: test/chemistry/mass_of_film_deposited_during_electrolysis_test.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    time = Quantity(1800 * units.second)
+
+    return Args(current=current,
+                molar_mass=molar_mass,
+                current_output=current_output,
+                valence=valence,
+                time=time)
+
+
+def test_basic_etching_rate(test_args: Args) -> None:
```
Comment (blackyblack) [inline test/chemistry/mass_of_film_deposited_during_electrolysis_test.py:28]: It's not etching rate.

File: test/chemistry/mass_of_film_deposited_during_electrolysis_test.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
+from collections import namedtuple
+from pytest import fixture, raises
+from symplyphysics import (assert_equal, units, Quantity, errors, prefixes)
+from symplyphysics.laws.chemistry import mass_of_film_deposited_during_electrolysis as mass_law
+
+# Description
+## The current is 10 milliampere. The molar mass of a metal atom is 63.546 gram per mole. The current output is 1.05, and the valence of the metal atom is 29.
```
Comment (blackyblack) [inline test/chemistry/mass_of_film_deposited_during_electrolysis_test.py:7]: How can be current output higher than 1? It is higher than input current?

Comment (Zulusssss) [reply test/chemistry/mass_of_film_deposited_during_electrolysis_test.py:7]: > How can be current output higher than 1? It is higher than input current?

The current output characterizes only the electrochemical process, i.e., for example, with the anodic dissolution of copper in a sulfuric acid electrolyte, the current output is close to 100%, but another 5% can be added due to the chemical dissolution of copper in the electrolyte. As a result, the calculated current output can formally be 105% due to the chemical dissolution of copper.

Comment (blackyblack) [reply test/chemistry/mass_of_film_deposited_during_electrolysis_test.py:7]: Add some explanation to the description, to avoid future users' confusion.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**